### DISCUSSION TITLE
好评！

### DISCUSSION THREAD

**Discussion by HoldOnBro** [kayanouriko/quantumultx-mihoyobbs-auto-helper#1] [General]
自己手动抓的用不了，还得是大佬做的脚本好使

**Comment by kayanouriko** [comment]
> 自己手动抓的用不了，还得是大佬做的脚本好使

我也是复制粘贴搬运工，感谢有大佬逆向出接口参数供我们使用。🥳

**Comment by Tangguoo** [comment]
米游社的社区签到不能使用了，大佬修复一下

**Reply by kayanouriko** [reply]
这是米游社后端的安全性设计，不可避免，无法绕过.
目前上游项目也没法解决, 因为目前风控解除的方案是需要付费的, 所以我一直没有修. 这样来看该脚本已经失去相对应的价值了.
推荐使用一个拥有复合型功能的应用(eg. [胡桃工具箱](https://hut.ao/))来管理米游社额外的功能.
